kitsune/wiki/jinja2/wiki/document.html

```diff
@@ -57,6 +57,13 @@
     {{ search_box(settings, id='support-search-sidebar', params=search_params) }}
   </div>
 
+  {% if in_staff_group(user) and document.is_linked_in_product %}
+  <figure id="linked-in-product-indicator">
```

Comment:
Instead of using `grid` for CSS I would go for something like:

```sass
figure {
  &.foobar {
    display: flex;
    align-items: center;
    justify-content: space-between;
    margin-bottom: p.$spacing-xl;
  }


  img {
    width: 24px;
    height: 24px;
  }

  figcaption {
    font-weight: 500;
    line-height: normal;
  }
}
```

Reply:
Regarding the SCSS, I used `flex` instead of `grid`, but not some of the other things, like `align-items: center;` and `justify-content: space-between;`, since those deviated from what Josh wanted.

Here's what it looked like with what you have above:
<img width="1249" alt="image" src="https://github.com/mozilla/kitsune/assets/3743693/94bf3302-46c0-48c6-8b06-eb7919c41b26">

Reply:
that was merely a suggestion, the main point was to use flexbox instead of grid for this